src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -119,13 +128,58 @@ def serialize(self, result, root, snuba_event, event_id=None):
 
             # For the light response, the parent will be unknown unless it is a direct descendent of the root
             trace_results.append(
-                serialize_event(snuba_event, root["id"] if root_span is not None else None)
+                self.serialize_event(
+                    snuba_event, root["id"] if root_span is not None else None, None
+                )
             )
 
         event = eventstore.get_event_by_id(snuba_event["project_id"], event_id)
         for span in event.data.get("spans", []):
             if span["span_id"] in parent_map:
                 child_event = parent_map[span["span_id"]]
-                trace_results.append(serialize_event(child_event, event_id))
+                trace_results.append(self.serialize_event(child_event, event_id, None))
 
         return trace_results
+
+
+class OrganizationEventsTraceEndpoint(OrganizationEventsTraceEndpointBase):
+    def serialize_event(self, *args, **kwargs):
+        event = super().serialize_event(*args, **kwargs)
+        event["children"] = []
```

Comment:
For the full trace view, we'll need to include the timings in the serialization.

Reply:
🤔 Not sure I understand what you mean, are you asking to wrap serialization in a span?

Reply:
Ah, sorry, I guess I wasn't clear. What I mean is in order to render a waterfall view of the trace, we need to include the start/end timestamps of all the transactions in the serialization.

Reply:
We discussed via slack, but definitely going to do this in a separate PR. we only expose `finish_ts` as `timestamp` via discover right now so we'd have to get `start_ts` and we'll want `start_ms` and `finish_ms` for the waterfall view too.